src/main/java/teammates/common/datatransfer/FeedbackQuestionAttributes.java

```diff
@@ -287,7 +287,7 @@ public boolean isResponseVisibleTo(FeedbackParticipantType userType) {
      * @param newAttributes
      * @return
      */
-    public boolean isChangesRequiresResponseDeletion(FeedbackQuestionAttributes newAttributes) {
+    public boolean doChangesRequireResponseDeletion(FeedbackQuestionAttributes newAttributes) {
```

Comment:
Unsure if this name is okay (tag @damithc for confirmation) as `do` is not part of the current [boolean naming conventions](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#specific-naming-conventions).

In the event that `do` is not acceptable, it might be possible to rewrite the method name as `areResponseDeletionsRequiredForChanges` instead.